src/implementations/vanilla/components/menu/menu.scss

```diff
@@ -85,6 +85,12 @@ $menu: null !default;
       display: block;
     }
 
+    // For the external icon
+    .ecl-menu__item .ecl-link__label + .ecl-icon {
```

Comment:
Is it coming from the designers? The icon is quite huge here...
![image](https://github.com/ec-europa/europa-component-library/assets/25579221/88605fc9-d594-4c8f-b51c-ce54fbd89a41)

Also it may be safer to use a css selector not directly linked to the markup structure, like
```
.ecl-menu__link .ecl-icon, 
.ecl-menu__sublink .ecl-icon {
  ...
}
```

Reply:
We have to override the size for the external icon when used in the first level menu items, the 2xs set by default by the link template makes it super small, the previous selector used was wrong, basically, targeting also the sublinks, i updated it now using the suggested .ecl-menu__link .ecl-icon.
I think the "m" size is fine for the large viewports, but it is an arbitrary choice, no indication by the designers.

Reply:
ok, let's wait for the designers' feedback